add-failedLoginsCount.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE staff
+ADD failedLoginsCount TINYINT DEFAULT '0';
```

Comment:
```suggestion
ADD failedLoginsCount TINYINT DEFAULT 0;
```

Seems to work either way, but just looks wrong FYI.